server/src/main/java/io/druid/initialization/Initialization.java

```diff
@@ -275,6 +278,41 @@ public static URLClassLoader getClassLoaderForExtension(File extension) throws M
     return loader;
   }
 
+  public static List<URL> getURLsForClasspath(String cp)
+  {
+    try {
+      String[] paths = cp.split(File.pathSeparator);
+
+      List<URL> urls = new ArrayList<>();
+      for (int i = 0; i < paths.length; i++) {
+        File f = new File(paths[i]);
+        if ("*".equals(f.getName())) {
```

Comment:
Yeah, I couldn't find a good standard way when i peeked around. For "normal" looking classpaths this shouldn't be an issue.